src/mlpack/core/cv/cv_base.hpp

```diff
@@ -0,0 +1,329 @@
+/**
+ * @file cv_base.hpp
+ * @author Kirill Mishchenko
+ *
+ * A base class for cross-validation.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_CV_CV_BASE_HPP
+#define MLPACK_CORE_CV_CV_BASE_HPP
+
+#include <mlpack/core/cv/meta_info_extractor.hpp>
+
+namespace mlpack {
+namespace cv {
+
+/*
```

Comment:
Unfortunately Doxygen won't grab these comments as documentation unless you start with two *'s: `/**`.